pkgs/python_ot/pyproject.toml

```diff
@@ -4,101 +4,73 @@
 
 [tool.poetry]
 name = "opentitan"
-version = "0.1.0"
+version = "0.0.0"
 description = "Open-source hardware root-of-trust"
 authors = []
-license = "Apache 2.0"
-readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.10"
-anytree = "2.8.0"
-attrs = "23.1.0"
-beautifulsoup4 = "4.12.2"
-blessed = "1.20.0"
-build = "0.10.0"
-cachetools = "5.3.1"
-certifi = "2023.7.22"
-charset-normalizer = "3.2.0"
-click = "8.1.7"
-commonmark = "0.9.1"
-cssselect = "1.2.0"
-cssutils = "2.7.1"
-cython = "3.0.1"
-distro = "1.8.0"
-enlighten = "1.10.2"
-flake8 = "5.0.4"
-gitdb = "4.0.10"
-gitpython = "3.1.32"
-hjson = "3.1.0"
-idna = "3.4"
-importlib-resources = "1.4.0"
-iniconfig = "2.0.0"
-isort = "5.10.1"
-jinja2 = "3.1.2"
-jsonschema = "4.17.3"
-libclang = "16.0.0"
-libcst = "0.4.1"
-libusb1 = "3.0.0"
-lizard = "1.17.10"
-lxml = "4.9.2"
-mako = "1.1.6"
-markupsafe = "2.1.3"
-mccabe = "0.7.0"
-mistletoe = "0.9.0"
-msgpack-python = "0.5.6"
-mypy = "0.971"
-mypy-extensions = "1.0.0"
-okonomiyaki = "1.4.0"
-packaging = "23.1"
-pkgutil-resolve-name = "1.3.10"
-pluggy = "1.2.0"
-pluralizer = "1.2.0"
-prefixed = "0.7.0"
-premailer = "3.8.0"
-py = "1.11.0"
-pycodestyle = "2.9.1"
-pycryptodome = "3.15.0"
-pydriller = "2.1"
-pyelftools = "0.29"
-pyfinite = "1.9.1"
-pyflakes = "2.5.0"
-pygments = "2.16.1"
-pyparsing = "3.1.1"
-pyproject-hooks = "1.0.0"
-pyrsistent = "0.19.3"
-pyserial = "3.5"
-pytest = "7.0.1"
-pytest-timeout = "2.1.0"
-pytz = "2023.3"
-pyyaml = "6.0"
-requests = "2.31.0"
-rich = "12.6.0"
-semantic-version = "2.10.0"
-simplesat = "0.8.2"
-six = "1.16.0"
-smmap = "5.0.0"
-soupsieve = "2.4.1"
-tabulate = "0.8.10"
-termcolor = "1.1.0"
-tomli = "2.0.1"
-typer = "0.6.1"
-types-dataclasses = "0.6.6"
-types-pkg-resources = "0.1.3"
-types-pytz = "2023.3.0.1"
-types-pyyaml = "6.0.11"
-types-tabulate = "0.8.11"
-typing-extensions = "4.7.1"
-typing-inspect = "0.9.0"
-urllib3 = "2.0.4"
-wcwidth = "0.2.6"
-wheel = "0.41.2"
-yapf = "0.32.0"
-zipfile2 = "0.0.12"
-chipwhisperer = { url = "https://github.com/newaetech/chipwhisperer-minimal/archive/2643131b71e528791446ee1bab7359120288f4ab.zip" }
-edalize = { url = "https://github.com/lowRISC/edalize/archive/refs/tags/v0.4.0.zip" }
-fusesoc = { url = "https://github.com/lowRISC/fusesoc/archive/refs/tags/ot-0.5.zip" }
+python = "^3.8.0"
 
-[build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+# Keep sorted.
```

Comment:
Maybe drop this comment? The sorting is … dubious :)

Reply:
What's not sorted?

Reply:
Only the first 15 lines are sorted

Reply:
This is how OT python-requirements.in does it

Reply:
If you want to match the comments in that file exactly, that’s fine too I guess.